src/concepts/pipeline/gcp_data_pipeline.md

```diff
@@ -0,0 +1,194 @@
+of the data using the `normalized_channel` and `sample_id` fields.
+
+# Beyond the Data Warehouse
+
+The diagram above shows the path data takes to get into the data warehouse.
+After that, we have to start using it!
+
+## Workflow Management and ETL
+
+We use [Airflow] for workflow management.
+
+It orchestrates the daily creation of the Stable tables described above,
+as well as many other derived datasets.
+
+The ETL code to create derived datasets is commonly implemented using queries in BigQuery.
+
+Many examples can be found in the [bigquery-etl] repository.
+
+Data in BigQuery is also accessible via Spark, and several ETL jobs also run via Dataproc.
+
+These jobs produce data sets that are used for downstream analysis and data
+applications (such as [measurement][TMO] and [stability][MC] dashboards,
+[addon recommendation][taar], and other [data products]).
+
+## Data Analysis
+
+Once the data reaches our data warehouse in BigQuery it can be processed
+in a number of ways as described in the [Accessing BigQuery] article.
+
+Data analysis is most commonly done using [SQL queries][stmo] or using [Spark].
+
+[collect data]: https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/collection/index.html
+[histograms]: https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/collection/histograms.html
+[scalars]: https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/collection/scalars.html
+[timings]: https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/collection/measuring-time.html
+[events]: https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/collection/events.html
+[probes]: ../../datasets/new_data.md
+[collection policy]: https://wiki.mozilla.org/Firefox/Data_Collection
+[subsessions]: https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/concepts/sessions.html#subsessions
+[main ping]: https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/data/main-ping.html
+[ping types]: https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/concepts/pings.html#ping-types
+[create their own ping types]: https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/collection/custom-pings.html
+[API]: https://dxr.mozilla.org/mozilla-central/rev/6a23526fe5168087d7e4132c0705aefcaed5f571/toolkit/components/telemetry/TelemetryController.jsm#202
+[submit]: https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/concepts/submission.html#submission
+[Airflow]: https://github.com/mozilla/telemetry-airflow/
+[TMO]: https://telemetry.mozilla.org/
+[re:dash]: https://sql.telemetry.mozilla.org/
+[STMO]: https://sql.telemetry.mozilla.org/
+[Spark]: ../../tools/spark.md
+[Accessing BigQuery]: ../../cookbooks/bigquery.md
+[taar]: https://github.com/mozilla/taar
+[MC]: https://data-missioncontrol.dev.mozaws.net
```

Comment:
Should be https://missioncontrol.telemetry.mozilla.org

Reply:
d'oh, awesomebar fail. will fix.